src/main/scala/outwatch/dom/VDomModifier.scala

```diff
@@ -46,15 +56,15 @@ final case class ChildStreamReceiver(childStream: Observable[VNode]) extends Chi
 final case class ChildrenStreamReceiver(childrenStream: Observable[Seq[VNode]]) extends ChildVNode
 
 sealed trait VNode_ extends ChildVNode {
-  // TODO: have apply() only on VTree?
```

Comment:
We need to keep this TODO!

Reply:
I updated the PR and removed the `apply()` from `VNode_`.